src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -382,6 +398,38 @@ public function getName()
         return 'request';
     }
 
+    public function collectSessionUsage(): void
+    {
+        $trace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
```

Comment:
Wouldn't this always get a backtrace to the AbstractsessionListener calling that method ? I fail to see how this is useful. This listener is not the place where the usage of the session happen (at least in my understanding of what "using the session" means).